public/app/features/playlist/api.ts

```diff
@@ -13,15 +13,24 @@ export async function updatePlaylist(id: number, playlist: Playlist) {
   await withErrorHandling(async () => await getBackendSrv().put(`/api/playlists/${id}`, playlist));
 }
 
+export async function deletePlaylist(id: number) {
+  await withErrorHandling(async () => await getBackendSrv().delete(`/api/playlists/${id}`), 'Playlist deleted');
```

Comment:
I think we can replace all usages of `async () => await getBackendSrv()` with `() => getBackendSrv()`

Reply:
yes, thank you!